### PR TITLE
BITMAKER-1840 estela: Configure repository descriptions

### PR DESCRIPTION
**Pull request by raymond1242**
We need to define the badges, links, description, and GitHub topics of estela repositories:
Consider all the github editable fields that offer a better understanding of the repositories

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,13 +2,21 @@
 
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 
+[![version](https://img.shields.io/badge/version-0.1-blue)](https://github.com/bitmakerla/estela)
+[![yarn-version](https://img.shields.io/badge/yarn-v1.22.19-blue)](https://yarnpkg.com)
+[![django-version](https://img.shields.io/badge/Django-v3.1.1-orange)](https://www.djangoproject.com)
+[![build](https://img.shields.io/badge/build-passing-brightgreen)](https://github.com/bitmakerla/estela/actions)
+![CI](https://github.com/eslint/eslint/workflows/CI/badge.svg)
+[![licene](https://img.shields.io/badge/license-MIT-lightgrey)](https://github.com/bitmakerla/estela/blob/main/LICENSE.md)
```
Comment (rodp63) [inline README.md:10]: change this to `license`

File: README.md [hunk 2]
```diff
@@ -2,13 +2,21 @@
...
+
+
 <h4 align="center">
-estela is an elastic web scraping cluster running on Kubernetes. It provides mechanisms to deploy, run and scale
+<strong>estela</strong> is an elastic web scraping cluster running on <a>Kubernetes</a>. It provides mechanisms to deploy, run and scale
```
Comment (rodp63) [inline README.md:14]: Remove the `<a> </a>` tag please

**Review by rodp63 — CHANGES_REQUESTED** [review]
Please, put all the badges in one line. The style badge should be at the end.

**Comment by raymond1242** [comment]
@joaquingx I already have fixed the issues

**Review by rodp63 — CHANGES_REQUESTED** [review]
Since everything is centered, I think the badges should be centered as well

**Review by rodp63 — APPROVED** [review]
Looks good :sparkles:

**Pull request merged**